src/main/java/school/lemon/changerequest/java/introduction/hw2/Task10.java

```diff
@@ -1,7 +1,26 @@
 package school.lemon.changerequest.java.introduction.hw2;
 
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 public class Task10 {
     public static String trim(String text) {
-        return "";
+        if (text == "") {
```

Comment:
you should not compare strings using `==` operator. If you want to check whether string is equal to another string - use `String.equals()` method.